### ISSUE TITLE
Test prov editor on all supported browsers

### ISSUE THREAD

**Issue by gothub** [NCEAS/metacatui#373]
The the prov editor on Chrome, Firefox, Safari, IE to verify rendering and all functionality.

**Comment by laurenwalker** [comment]
@gothub Has this testing been completed already? The prov editor will be going into production soon, as it is in beta now.

**Comment by gothub** [comment]
This issue has me as the assignee, but really there are several people that need to weigh in on this, including the dev team, the data team, data fellows, etc, now that there are 30+ datasets in production that have prov included. 

The question to me is, how do we coordinate input from all interested parties to say that the prov editor works on all supported platforms?

PS - I'll do some verification testing tomorrow, so I can register my vote of 'doneness'.

**Comment by laurenwalker** [comment]
When we did the metadata editor testing, we included a task involving the prov editor, so the data team has already done a bit of prov editor testing on all browsers.

For the metadata editor testing, I created a short list of tasks to complete in the editor and made sure that each task was done in every supported browser (Chrome, FF, Safari, IE 11, and Edge), and in both Windows and Mac. I think that could be a sufficient testing route to take for the prov editor as well.

I coordinated with Jesse to get the data team on board for completing those testing tasks. It was easiest if I created a spreadsheet for them and then let them fill in the cells as they tested: https://docs.google.com/spreadsheets/d/1YLPMkzEP3g3-j8ud2aJv_fMitGFMSaPygsy9trmCWvY/edit#gid=1652317615
